src/main/java/com/microsoft/jenkins/containeragents/aci/AciService.java

```diff
@@ -45,13 +45,20 @@
     private static final String DEPLOY_TEMPLATE_FILENAME
             = "/com/microsoft/jenkins/containeragents/aci/deployTemplate.json";
 
+    private static final String DEPLOY_TEMPLATE_WITH_LOG_ANALYTICS_FILENAME
```

Comment:
Prefer modifying the arm template via json manipulation, otherwise we end up with ~10 variations of the file like in azure-vm-agents.

this sort of thing:
https://github.com/jenkinsci/azure-vm-agents-plugin/blob/75f3235b72021daa6ae0e5fd415d3c7652b394bd/src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java#L386